http2/src/main/java/com/blazemeter/jmeter/http2/visualizers/ResultCollector.java

```diff
@@ -1,57 +0,0 @@
-package com.blazemeter.jmeter.http2.visualizers;
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import org.apache.jmeter.reporters.Summariser;
-import org.apache.jmeter.samplers.SampleEvent;
-import org.apache.jmeter.samplers.SampleResult;
-import com.blazemeter.jmeter.http2.sampler.HTTP2SampleResult;
-
-public class ResultCollector extends org.apache.jmeter.reporters.ResultCollector {
```

Comment:
What will happen after update if user already have this element in TestPlan?
The same question about SumpleDataWriter